app/schema/api/v3/models.rb

```diff
@@ -154,8 +154,8 @@ def blood_sugar
       { type: :object,
         properties: {
           id: { '$ref' => '#/definitions/uuid' },
-          blood_sugar_type: { type: :string, enum: BloodSugar.blood_sugar_types.keys },
-          blood_sugar_value: { type: :integer },
+          blood_sugar_type: { type: :string, enum: BloodSugar.blood_sugar_types.keys.reject { |type| type == 'hba1c' } },
+          blood_sugar_value: { type: :number },
```

Comment:
Should this go back to `integer` now?  Technically a datatype change could be a breaking change, right?

Reply:
Yes, adding a spec for this